spec/features/peep_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'peep'
+
+describe Peep do
+  it { is_expected.to have_property :id }
+  it { is_expected.to have_property :name }
+  it { is_expected.to have_property :user_name }
+
+end
```

Comment:
This is not a feature spec, rather a unit or model spec (whatever you prefer). It should not be placed in the `features` folder.